katharsis-core/src/main/java/io/katharsis/core/internal/boot/KatharsisBoot.java

```diff
@@ -349,6 +348,10 @@ public String getWebPathPrefix() {
 		return getProperty(KatharsisProperties.WEB_PATH_PREFIX);
 	}
 
+	public boolean isNullDataResponseEnabled() {
+		return Boolean.parseBoolean(getProperty(KatharsisProperties.NULL_DATA_RESPONSE_ENABLED));
```

Comment:
I looked at this `getProperty` method, and it can/should be moved to either and abstract class do a default method in the interface. 

Also (and this isn't your fault) - That whole properties provider class is a total replication of functionality that comes with spring by default.